Buildings/Resources/Scripts/Dymola/Fluid/HeatExchangers/Examples/WetCoilEffectivenessNTUMassFlow.mos

```diff
@@ -1,11 +1,8 @@
-old_StoreProtectedVariables = Advanced.StoreProtectedVariables;
-Advanced.StoreProtectedVariables = true;
```

Comment:
@mwetter This was needed to plot the dry fraction `hexWetNTU.dryWetCalcs.dryFra` and illustrate that the validation case covers all the conditions from fully dry to fully wet (and originally to compare with the dry fraction computed by `WetCoilCounterFlow` but I see you removed that addition as it triggered an event). 
Even if it is not used for validation any more, I think plotting `hexWetNTU.dryWetCalcs.dryFra` still makes sense to illustrate what the validation case covers.

Reply:
@AntoineGautier : Please see https://github.com/lbl-srg/modelica-buildings/pull/2417